.golangci.yml

```diff
@@ -188,3 +189,50 @@ issues:
         - gosec
       text: "G404:"
       path: private/bufpkg/bufimage/bufimageutil/.*_test\.go
+    # This test shouldn't run in parallel as it allocates a lot of memory.
+    - linters:
+        - paralleltest
+      text: "missing the call to method parallel"
+      path: private/bufpkg/bufimage/bufimagebuild/builder_unix_test.go
+    # Every test in this file calls one of testLintConfigModifier and testLint,
+    # both of which call t.Parallel().
+    - linters:
+        - paralleltest
+      text: "missing the call to method parallel"
+      path: private/bufpkg/bufcheck/buflint/buflint_test.go
+    # TestConfigInclusion and TestLicense already call T.parallel(), but the other
+    # tests in this file calls one of testBucketGetFileInfos, testBucketGetAllFileInfosError,
+    # testBucketGetFileInfosForExternalPathsError, and testDocumentationBucket,
+    # all of which call t.Parallel().
+    - linters:
+        - paralleltest
+      text: "missing the call to method parallel"
+      path: private/bufpkg/bufmodule/bufmodulebuild/module_bucket_builder_test.go
+    # Every test in this file calls testRunLint, which calls t.Parallel().
+    - linters:
+        - paralleltest
+      text: "missing the call to method parallel"
+      path: private/buf/cmd/protoc-gen-buf-lint/lint_test.go
+    # Every test in this file calls testBreaking, which calls t.Parallel().
+    - linters:
+        - paralleltest
+      text: "missing the call to method parallel"
+      path: private/bufpkg/bufcheck/bufbreaking/bufbreaking_test.go
+    # Every test in this file calls testRoundTripLocalFile, which calls t.Paralell().
+    - linters:
+        - paralleltest
+      text: "missing the call to method parallel"
+      path: private/buf/buffetch/buffetch_test.go
+    # TestWorkspaceArchiveDir, TestWorkspaceWithInvalidArchivePathFail
+    # and TestWorkspaceWithInvalidArchiveAbsolutePathFail cannot run in parallel
+    # because they all call createZipFromDir on the same path.
+    - linters:
+        - paralleltest
+      text: "missing the call to method parallel"
+      path: private/buf/cmd/buf/workspace_test.go
+    # Parallelizing TestPlainPostHandlerTLS and TestPlainPostHandlerH2C
+    # makes this test flaky.
+    - linters:
+        - paralleltest
+      text: "missing the call to method parallel"
```

Comment:
Why do we need to match on text here? I think this can be removed

Reply:
This question wasn't resolved, please don't mark as resolved unless it is.

Reply:
This linter also has other checks, such as [tc variable reinitialised](https://github.com/kunwardeep/paralleltest#tparallel-is-called-in-the-range-method-and-test-case-variable-tc-being-used-but-is-not-reinitialised-more-info), ensuring loop variable doesn't get captured by `t.Run`. The text is here to only exempt this specific check for this file.

Sorry about the resolution